calc/calc_graph_fv_test.go

```diff
@@ -357,6 +354,17 @@ var baseTests = []StateList{
 		hostInIPPool,
 		vxlanWithBlock,
 	},
+	{
+		// Corner case: hosts with duplicate IPs.
+		vxlanWithBlock,
+		vxlanWithBlockDupNodeIP,
+		vxlanWithBlock,
+	},
+	{
+		// Corner case: hosts with duplicate IPs.
```

Comment:
nit: would it help to differentiate this corner case comment from the immediately preceding one?